plugins/circleci-deploy/.toolkitrc.yml

```diff
@@ -71,6 +71,10 @@ options:
                   !toolkit/if-defined '@dotcom-tool-kit/serverless.awsAccountId':
                     aws-account-id: !toolkit/option '@dotcom-tool-kit/serverless.awsAccountId'
                     system-code: !toolkit/option '@dotcom-tool-kit/serverless.systemCode'
+                  !toolkit/if-defined '@dotcom-tool-kit/cloudsmith.organisation':
```

Comment:
**issue**: I've tested and confirmed that Tool Kit will _not_ let you set options for plugins that don't exist (to help catch typos) so we'll have to make a barebones Tool Kit plugin for Cloudsmith.